app/Actions/PrepareStreams.php

```diff
@@ -8,12 +8,9 @@
 
 class PrepareStreams
```

Comment:
Since this action is now only used to sort the stream, let's rename it to `SortStreamsByDateAction`.